zerver/lib/webhooks/git.py

```diff
@@ -209,7 +210,14 @@ def get_pull_request_event_message(
         assignee_info = PULL_REQUEST_OR_ISSUE_ASSIGNEE_INFO_TEMPLATE.format(assignee=assignee)
         main_message = f"{main_message} {assignee_info}"
 
-    if target_branch and base_branch:
+    if target_branch and base_branch and should_use_branch_info_template:
+        branch_info = PULL_REQUEST_BRANCH_INFO_TEMPLATE.format(
+            target=target_branch,
+            base=base_branch,
+        )
+        main_message = f"{main_message} {branch_info}"
+
+    elif target_branch and base_branch and should_use_branch_info_template is None:
```

Comment:
The bodies of these are two conditionals are doing the same thing, right? (Or is there a subtle difference I'm missing?)

In that case, best to consolidate them into one. The condition could be:
```py
 if target_branch and base_branch and should_use_branch_info_template != False:
```

but better yet, this parameter could have a default of `True` up in the function definition, and then the condition is just:
```py
 if target_branch and base_branch and should_use_branch_info_template:
```

Reply:
This piece of code has been removed with the latest commit.